.github/compose/go-fdo.yaml

```diff
@@ -0,0 +1,87 @@
+services:
+
+  manufacturer:
+    container_name: manufacturer
+    hostname: manufacturer
+    image: manufacturer
+    build: https://github.com/fido-device-onboard/go-fdo-server.git#main
+    environment:
+      TZ: Europe/Madrid
+    ports:
+      - 8038:8038
+    # healthcheck:
+    #   test: ["CMD-SHELL", "test $(curl --silent --output /dev/null --write-out '%{http_code}' http://localhost:8038/health) = 200" ]
+    #   interval: 2s
+    #   timeout: 5s
+    #   retries: 3
+    #   start_period: 1s
+    command: --db=/tmp/sqlite.db --db-pass 'Redh@t#1' --debug serve manufacturer:8038
+    networks:
+      - fdo
+    restart: unless-stopped
+
+  rendezvous:
+    container_name: rendezvous
+    hostname: rendezvous
+    image: rendezvous
+    build: https://github.com/fido-device-onboard/go-fdo-server.git#main
+    environment:
+      TZ: Europe/Madrid
+    ports:
+      - 8041:8041
+    command: --db=/tmp/sqlite.db --db-pass 'Redh@t#1' --debug serve rendezvous:8041
+    networks:
+      - fdo
+    restart: unless-stopped
+
+  owner:
+    container_name: owner
+    hostname: owner
+    image: owner
+    build: https://github.com/fido-device-onboard/go-fdo-server.git#main
+    environment:
+      TZ: Europe/Madrid
+    ports:
+      - 8043:8043
+    command: --db=/tmp/sqlite.db --db-pass 'Redh@t#1' --debug serve owner:8043
+    networks:
+      - fdo
+    restart: unless-stopped
+
+  client:
```

Comment:
@mmartinv Question: are the "client" and "client-onboarding" containers used by the workflow?  IIUC the workflow does a docker build as well.